wavepool/models.py

```diff
@@ -38,3 +38,7 @@ def tags(self):
         return [
             'HR', 'Diversity & Inclusion', 'Culture'
         ]
+
+    @property
+    def edit_url(self):
+        return reverse('admin:wavepool_newspost_change', args=[self.pk])
```

Comment:
Here i Introduced a `edit_url` property that is eventually used in `newspost.html` to surface the edit URL to CMS users